components/event-processor/org.wso2.carbon.event.processor.core/src/main/java/org/wso2/carbon/event/processor/core/EventProcessorService.java

```diff
@@ -181,7 +181,7 @@ public void setStatisticsEnabled(String executionPlanName, boolean isEnabled,
      * @param executionPlan execution plan.
      * @return  true if valid.
      */
-    public boolean validateExecutionPlan(String executionPlan);
+    public String validateExecutionPlan(String executionPlan);
```

Comment:
As discussed with Suho offline, did the refactoring as per the following commmit: 
https://github.com/dilini-muthumala/carbon-event-processing/commit/fa408b27c40a3872ddeb19bef1c32e48c8fd06c3